### PR TITLE
Add info about subtheming

### PR DESCRIPTION
**Pull request by cnovak**
Info about sidebars coming in 1.1

### CODE REVIEW

**Review by jacine — APPROVED** [review]
Made some minor adjustments.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The core functionality of this profile is in active development. The features an
 
 ## Planned features
 
+* **Sidebar regions:** Two sidebar block regions will be added to the `apigee_kickstart` theme, included in this profile, in the 8.x-1.1 release. It is recommended to wait for the 8.x-1.1 release before creating a subtheme, if you intend to make changes to the page or node templates, in order to reduce rework.
```
Comment (cnovak) [inline README.md]:
```suggestion
* **Sidebar regions:** Two sidebar block regions will be added to the `apigee_kickstart` theme, included in this profile, in the 8.x-1.1 release. It is recommended to wait for the 8.x-1.1 release before creating a subtheme if you intend to make changes to the page or node templates in order to reduce rework.
```

File: themes/custom/apigee_kickstart/README.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-#### Step 3
+### Step 3
 Update `proxy` in **webpack.mix.json**.
 
-#### Step 4
+### Step 4
 Run the following command to compile Sass and watch for changes: `npm run watch`.
+
+## Customizing this theme
+It is recommended to [create a subtheme](https://www.drupal.org/docs/8/theming-drupal-8/creating-a-drupal-8-sub-theme-or-sub-theme-of-sub-theme) in order to [customize this theme](https://www.drupal.org/docs/8/theming). Note: Sidebar block regions will be added to this theme in the 8.x-1.1 release. It is recommended to wait for the 8.x-1.1 release before creating a subtheme, if you intend to make changes to the page or node templates, in order to reduce rework.
```
Comment (cnovak) [inline themes/custom/apigee_kickstart/README.md]:
```suggestion
It is recommended to [create a subtheme](https://www.drupal.org/docs/8/theming-drupal-8/creating-a-drupal-8-sub-theme-or-sub-theme-of-sub-theme) in order to [customize this theme](https://www.drupal.org/docs/8/theming). Note: Sidebar block regions will be added to this theme in the 8.x-1.1 release. It is recommended to wait for the 8.x-1.1 release before creating a subtheme if you intend to make changes to the page or node templates in order to reduce rework.
```

**Comment by cnovak** [comment]
@jacine and I are both authors on PR, both of us have signed CLA

**Pull request merged**